docs/guides/setup/installation/components_local/_check_python_version.mdx

```diff
@@ -0,0 +1,21 @@
+<!--
+
+---Import---
+import CheckPythonVersion from './_check_python_version.mdx'
+<CheckPythonVersion />
+
+---Header---
+### 1. Check Python version
+-->
+
+First, check that you have Python 3 installed. You can check your version of Python by running:
+
+```console title="Terminal command"
+python --version
+```
+
+If this command returns something other than a Python 3 version number (like Python 3.X.X), you may need to try this:
+
+```console title="Terminal command"
+python3 --version
+```
```

Comment:
Maybe worth adding an admonition re "as of this writing, GE requires a minimum version 3.x.x for python" but you know written good. Though that would add the annoyance of having to change yet another file every time GE bumps version support.